src/hotspot/share/runtime/arguments.cpp

```diff
@@ -504,6 +504,7 @@ static SpecialFlag const special_jvm_flags[] = {
   { "DynamicDumpSharedSpaces",      JDK_Version::jdk(18), JDK_Version::jdk(19), JDK_Version::undefined() },
   { "RequireSharedSpaces",          JDK_Version::jdk(18), JDK_Version::jdk(19), JDK_Version::undefined() },
   { "UseSharedSpaces",              JDK_Version::jdk(18), JDK_Version::jdk(19), JDK_Version::undefined() },
+  { "MetaspaceReclaimPolicy",       JDK_Version::jdk(21), JDK_Version::undefined(), JDK_Version::undefined() },
```

Comment:
What you have done isn't deprecation, it is obsoletion - the flag is accepted but completely ignored (which is fine by me for a diagnostic flag). The above should be changed accordingly, or else continue to parse the flag and check for a valid value.

Reply:
If the flag only has one valid value now then it is effectively obsolete, so you may as well mark it as such. Existing users of the flag will get a warning either way.

Reply:
Okay, done so.